config/azkaban/exec-server/start-exec.sh

```diff
@@ -2,7 +2,7 @@
 
 script_dir=$(dirname $0)
 
-echo "Creating tables in database if they do not exist using create-all-sql.sql"
-mysql -h $DB_HOST -u$DB_USERNAME -p$DB_PASSWORD $DB_NAME < /azkaban-db/create-all-sql-0.1.0-SNAPSHOT.sql
+# echo "Creating tables in database if they do not exist using create-all-sql.sql"
+# mysql -h $DB_HOST -u$DB_USERNAME -p$DB_PASSWORD $DB_NAME < /azkaban-db/create-all-sql-0.1.0-SNAPSHOT.sql
```

Comment:
Should these be left in commented?

Reply:
I'm not confident in it, I'm happy to remove it.